src/actions/resetActions/changePasswordActions.js

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+import { CHANGE_PASSWORD } from '../types';
+import Notify from '../../utils/Notify';
+
+const changePasswordActions = (newpassword, confirmpassword, token) => (dispatch) => {
+  const body = {
+    user: { password: newpassword, confirm_password: confirmpassword },
```

Comment:
please remove the snake casing,

Reply:
let me do that

Reply:
You didn't implement the feedback properly. You can read more on camel casing here https://en.wikipedia.org/wiki/Camel_case

Reply:
noted